src/components/common/forms/form-field/form-field.js

```diff
@@ -0,0 +1,34 @@
+import WebComponent, { Component} from '#WebComponent';
+import css from './form-field.css?inline';
+
+
+export default Component ({
+    tagName: 'form-field',
+    styleCSS: css
+},
+
+class FormField extends WebComponent {
+    get value() {
+        return this._getDOM().querySelector('input').value;
+    }
+
+    render() {
+        const error = this.getAttribute('error') || '';
+        const rl = this.getAttribute('rl') || 'input';
+        const text = this.getAttribute('text') || 'text';
```

Comment:
Valor por defecto: No pondría "text" como tal... Si no ponemos text, queremos que esté vacío, por lo cual pondría cadena vacía:

```suggestion
        const text = this.getAttribute('text') || '';
```

Reply:
Añado a esto: que es r1? mejor nomenclatura plis